app/src/main/java/com/x8bit/bitwarden/data/platform/manager/BridgeServiceManagerImpl.kt

```diff
@@ -0,0 +1,72 @@
+package com.x8bit.bitwarden.data.platform.manager
+
+import android.content.Intent
+import com.bitwarden.bridge.IBridgeService
+import com.bitwarden.bridge.IBridgeServiceCallback
+import com.bitwarden.bridge.model.EncryptedAddTotpLoginItemData
+import com.bitwarden.bridge.model.SymmetricEncryptionKeyData
+import com.bitwarden.bridge.model.SymmetricEncryptionKeyFingerprintData
+import com.x8bit.bitwarden.data.platform.manager.model.FlagKey
+
+/**
+ * Default implementation of [BridgeServiceManager].
+ */
+class BridgeServiceManagerImpl(
+    private val featureFlagManager: FeatureFlagManager,
+) : BridgeServiceManager {
+
+    override val binder: IBridgeService.Stub?
+        // TODO: Check for Android API level as well: BITAU-102
+        get() {
+            return if (featureFlagManager.getFeatureFlag(FlagKey.AuthenticatorSync)) {
+                defaultBinder
+            } else {
+                // If the feature flag is not enabled, return a null binder which will no-op all
+                // service calls.
+                null
+            }
+        }
+
+    /**
+     * Default implementation of the bridge service binder.
+     */
+    private val defaultBinder = object : IBridgeService.Stub() {
+        override fun getVersionNumber(): String {
+            // TODO: BITAU-104
+            return ""
+        }
+
+        override fun checkSymmetricEncryptionKeyFingerprint(
+            data: SymmetricEncryptionKeyFingerprintData?,
```

Comment:
Should this be nullable?

Reply:
This is nullable because the AIDL stuff generates a java interface and so therefore don't have nullability info... 

Technically we could feel safe enough to make this (and others) non-nullable, but I'm in favor of just leaving the nullability here to keep this as super duper safe as possible... 

What do you think?